gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -52,8 +57,18 @@
 
   public static final String DEFAULT_PROJECT_ID = "projectid1";
   public static final int PORT = 8080;
-  private static final String GCD = "gcd-head";
-  private static final String GCD_LOC = '/' + GCD + ".zip";
+  private static final String GCD_VERSION = "gcd-v1beta2-rev1-2.1.2b";
```

Comment:
Would it be less confusing if I rename GCD_VERSION TO GCD? I am using the value (without the ".zip" as the name of the folder I am going to extract the archive and run from).